src/main/java/core/basesyntax/strategy/impl/PurchaseTransactionService.java

```
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.strategy.TransactionService;
+
+public class PurchaseTransactionService implements TransactionService {
+    public void makeTransaction(String fruits, int number) {
+        Storage.fruits.replace(fruits, Storage.fruits.get(fruits) - number);
```

lets throw an exception if there is not enough fruits